server/ladders-remote.ts

```diff
@@ -81,6 +83,9 @@ export class LadderStore {
 			} else {
 				room.add(`||Ladder (probably) updated, but score could not be retrieved (${error.message}).`);
 			}
+			// Clear mmrCache for the format to get the users updated rating next search
+			if (p1 && p1.mmrCache[formatid]) delete p1.mmrCache[formatid];
```

Comment:
The test that `p1.mmrCache[formatid]` exists is redundant; delete is a no-op in this case